### DISCUSSION TITLE
Doesn't spack update the env variable LD_LIBRARY_PATH?

### DISCUSSION THREAD

**Discussion by giuseppe998e** [spack/spack#36192] [Q&A] [answered]
Hi, I installed `mpich` using Spack, but I noticed that the latter does not update the `LD_LIBRARY_PATH` environment variable by adding the `mpich` library path when I load the package.  
How can I fix this?

**Comment by tgamblin** [comment, marked as answer]
See the note here in the v0.19 release: https://github.com/spack/spack/releases/tag/v0.19.0:

> `LD_LIBRARY_PATH` is no longer set by default by `spack load` or module loads.
> 
> Setting `LD_LIBRARY_PATH` in Spack environments/modules can cause binaries from
> outside of Spack to crash, and Spack's own builds use `RPATH` and do not need
> `LD_LIBRARY_PATH` set in order to run. If you still want the old behavior, you
> can run these commands to configure Spack to set `LD_LIBRARY_PATH`:
> 
> ```console
> spack config add modules:prefix_inspections:lib64:[LD_LIBRARY_PATH]
> spack config add modules:prefix_inspections:lib:[LD_LIBRARY_PATH]
> ```

**Reply by haampie** [reply]
This question gets asked so often, it would be good if googling `spack` and `LD_LIBRARY_PATH` would show the answer as a first hit
